<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FABSettings</class>
 <widget class="QWidget" name="FABSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>350</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="leftMargin">
    <number>6</number>
   </property>
   <property name="topMargin">
    <number>6</number>
   </property>
   <property name="rightMargin">
    <number>6</number>
   </property>
   <property name="bottomMargin">
    <number>6</number>
   </property>
   <item>
    <layout class="QGridLayout" name="gridLayout">
     <item row="5" column="0">
      <widget class="QLabel" name="lblButtonRole">
       <property name="text">
        <string>Button role</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QCheckBox" name="chkDisabled">
       <property name="text">
        <string>Disabled</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="lblRippleStyle">
       <property name="text">
        <string>Ripple style</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="cbxButtonRole">
       <item>
        <property name="text">
         <string>Default</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Primary</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Secondary</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="lblCorner">
       <property name="text">
        <string>Corner</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1">
      <widget class="QComboBox" name="cbxRippleStyle">
       <item>
        <property name="text">
         <string>Centered</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Positioned</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>No Ripple</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="sbxHorizontalOffset"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="lblHorizontalOffset">
       <property name="text">
        <string>Horizontal offset</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="sbxVerticalOffset"/>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="cbxCorner">
       <item>
        <property name="text">
         <string>Top Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Top Right</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bottom Left</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Bottom Right</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="lblVerticalOffset">
       <property name="text">
        <string>Vertical offset</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QCheckBox" name="chkMini">
       <property name="text">
        <string>Mini</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="MaterialFloatingActionButton" name="fab">
     <property name="icon">
      <iconset resource="../../resources/library/qtmaterialwidgets.qrc">
       <normaloff>:/icons/toggle/svg/production/ic_star_24px.svg</normaloff>:/icons/toggle/svg/production/ic_star_24px.svg</iconset>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MaterialFloatingActionButton</class>
   <extends>MaterialRaisedButton</extends>
   <header location="global">QtMaterialWidgets/fab.hpp</header>
  </customwidget>
  <customwidget>
   <class>MaterialFlatButton</class>
   <extends>QPushButton</extends>
   <header location="global">QtMaterialWidgets/flatbutton.hpp</header>
  </customwidget>
  <customwidget>
   <class>MaterialRaisedButton</class>
   <extends>MaterialFlatButton</extends>
   <header location="global">QtMaterialWidgets/raisedbutton.hpp</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../resources/library/qtmaterialwidgets.qrc"/>
 </resources>
 <connections/>
</ui>
